contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -209,7 +209,21 @@ abstract contract WithdrawalQueue is AccessControlEnumerable, PausableUntil, Wit
         uint256 hint;
     }
 
-    /// @notice Claim withdrawals batch once finalized (claimable).
+    /// @notice Return batch of claimable eth amounts that is locked for each request
+    /// @param _claimWithdrawalInputs list of withdrawal request ids and hints to claim
+    function getClaimableEther(ClaimWithdrawalInput[] calldata _claimWithdrawalInputs)
```

Comment:
Just a consideration: integrations would need to repack the args to ClaimWithdrawalInput manually
though, they could have passed two arrays without additional conversions in the middle

Reply:
Reasonable point. Will remove the structure.